.codeclimate.yml

```diff
@@ -33,18 +33,18 @@ checks:
     config:
       threshold: 4
   similar-code:
-    enabled: true
+    enabled: false
```

Comment:
I think we need this restriction because it's help us to write better code

Reply:
Thanks,I have  solved that issues related to codeclimate